rkt/rkt.go

```diff
@@ -189,11 +198,43 @@ func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
 	return aTabOut
 }
 
+func startProfile() (cpufile *os.File, memfile *os.File, err error) {
+	if globalFlags.CPUProfile != "" {
+		cpufile, err = os.Create(globalFlags.CPUProfile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("cannot create cpu profile file %q: %v", globalFlags.CPUProfile, err)
+		}
+		pprof.StartCPUProfile(cpufile)
+	}
+	if globalFlags.MemProfile != "" {
+		memfile, err = os.Create(globalFlags.MemProfile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("cannot create memory profile file %q: %v", globalFlags.MemProfile, err)
+		}
+	}
+	return cpufile, memfile, nil
+}
+
+func stopProfile(cpuprofile, memprofile *os.File) {
+	pprof.StopCPUProfile()
```

Comment:
wrap in either `nil` or flag checks in case they weren't enabled? I think `WriteHeapProfile` might do work whenever it's called, so we should avoid calling it if it's not enabled.